geos: link to docs if GEOS is not installed

otan opened the pull request:
Release note (sql change): Introduce a hint when GEOS is improperly
installed to the docs instructions on installing CockroachDB.

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/57004)
<!-- Reviewable:end -->

otan committed:
geos: link to docs if GEOS is not installed

Release note (sql change): Introduce a hint when GEOS is improperly
installed to the docs instructions on installing CockroachDB.

rytaft reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> Reviewed 3 of 3 files at r1.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57004#-:-MMpHUrH3I1juVJ5yZIh:bg7igkv)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @sumeerbhola)



<!-- Sent from Reviewable.io -->

sumeerbhola reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/> 

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/57004#-:-MMpYh7QC7yxXGcXoiTy:bg7igkv)* status: :shipit: complete! 2 of 0 LGTMs obtained (waiting on @sumeerbhola)



<!-- Sent from Reviewable.io -->

otan commented:
thanks!

bors r=rytaft,sumeerbhola
